components/image/image.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="/doc.css">
+    <link rel="stylesheet" href="/components/image/image.css">
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
+    <title>Avatar</title>
+</head>
+<body>
+
+ <nav>
+    <logo class="logo">BuildUI</logo>
+       <ul class="nav-links">
+           <li><a href="#">HOME</a></li>
+           <li><a href="#">DOCUMENTATION</a></li>
+       </ul>
+    </nav>
+
+    <input type="checkbox" id="check">
+    <label for="check"><i class="fas fa-bars" id="btn"></i><i class="fas fa-times" id="cancel"></i></label>
+   
+  <div class="side-drawer">
+    <ul>
+        <header>Components</header>
+        <li><a href="/components/html/alert.html">Alert</a></li>
+        <li><a href="/components/html/avatar.html">Avatar</a></li>
+        <li><a href="/components/html/badge.html">Badge</a></li>
+        <li><a href="/components/html/button.html">Button</a></li>
+        <li><a href="/components/html/card.html">Card</a></li>
+        <li><a href="#">Image</a></li>
+        <li><a href="/components/html/input.html">Input</a></li>
+        <li><a href="/components/html/text-ut.html">Text Utilities</a></li>
+        <li><a href="/components/html/list.html">Lists</a></li>
+        <li><a href="/components/html/nav.html">Navigation-Desktop</a></li>
+        <li><a href="/components/html/modal.html">Modal</a></li>
+        <li><a href="/components/html/rating.html">Rating</a></li>
+        <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+        <li><a href="/components/html/card.html">Simplified Grid</a></li>
+    </ul>
+  </div>
+
+    <div class="Component-container">
+
+        <div class="text  mt2" id="image">
+            <h2 class="pb1 h2 comp-title">Images</h2><br>
```

Comment:
there is no use of <br/> here ig

Reply:
i have written its css into doc.css file ,thankyou for the review